packages/bits/demo/src/components/demo/expander/expander-test/expander-test.component.html

```diff
@@ -0,0 +1,37 @@
+<div class="container">
+    <div class="row">
+        <div class="col">
+            <nui-expander id="nui-demo-expander-basic">
```

Comment:
I think the same id's in the original docs examples that these were pulled from can safely be removed since they were only there for testing purposes. Removing the unneeded id's would help clean up the examples a little bit. This one for example: 

https://github.com/solarwinds/nova/blob/f85ab1e22d20ac96cf786ec8be67ed159fffa6df/packages/bits/demo/src/components/demo/expander/expander-basic/expander-basic.example.component.html#L1

Reply:
It might be good to do a quick search in the code base for each of them to make sure I'm right.